### ISSUE TITLE
Tidal disapeared from My Apps

### ISSUE THREAD

**Issue by heniec** [nikclayton/android-squeezer#623]
After updating Squeezer to v 2.0.0, Tidal disapeared from My Apps.
Previous version was ok.
I'm connecting to my local Daphile server, has Tidal installed on mysqueezebox.com account connected to Daphile, everything seems to be configured properly, cause I can see and use Tidal in My Apps in Daphile web ui.
Being conencted in Squeezer to my server I see Spotty and Youtube in My Apps, but there is no Tidal.
Interesting - when I use Search in Squeezer main menu, returned results include Tidal.

**Comment by przemnet** [comment]
Hello,

I have the same problem.

I'm using DietPi with Squeeezbox/Squeezlite and after Squeezer upgrade I don't see TIDAL in My Apps- it was working fine before.

When I enter My Apps, I get message:
"This player is already registered to another user. If it was purchased frome someone, ask them to delete the player from their account and try again. If not, please visit www.mysqueezebox.com/support for assistance".

When I enter on Squeezer App Galery -> Featured Apps -> TIDAL I get message: "TIDAL is already installed".

TIDAL is visible in My apps when I enter Squeezbox server via Web Browser and it is also visible when I use another app for Android- Squeeze Controller.

After I start to play album from TIDAL with Web Browser or Squeeze Controller and next I open Squeezer, I see currently played song and the playlist.

Can you fix this problem?

Thanks in advance.

Looking forward,
Przemek

**Comment by el-bro** [comment]
Hi there,
I have exact same issue described by @przemnet 

Cheers,
lbr

> 
> 
> Hello,
> 
> I have the same problem.
> 
> I'm using DietPi with Squeeezbox/Squeezlite and after Squeezer upgrade I don't see TIDAL in My Apps- it was working fine before.
> 
> When I enter My Apps, I get message:
> "This player is already registered to another user. If it was purchased frome someone, ask them to delete the player from their account and try again. If not, please visit [www.mysqueezebox.com/support](http://www.mysqueezebox.com/support) for assistance".
> 
> When I enter on Squeezer App Galery -> Featured Apps -> TIDAL I get message: "TIDAL is already installed".
> 
> TIDAL is visible in My apps when I enter Squeezbox server via Web Browser and it is also visible when I use another app for Android- Squeeze Controller.
> 
> After I start to play album from TIDAL with Web Browser or Squeeze Controller and next I open Squeezer, I see currently played song and the playlist.
> 
> Can you fix this problem?
> 
> Thanks in advance.
> 
> Looking forward,
> Przemek
> 
> ![squeezer_My_Apps](https://user-images.githubusercontent.com/47325817/75494575-de81e580-59bc-11ea-8964-f9bf350f3f10.jpg)
> ![squeezer_App_Galery](https://user-images.githubusercontent.com/47325817/75494580-df1a7c00-59bc-11ea-8c56-d96c6f72572e.jpg)
> ![squeezer_web_browser](https://user-images.githubusercontent.com/47325817/75494584-dfb31280-59bc-11ea-97ba-715332e80cc4.png)

**Comment by j-ame-s** [comment]
I have some more info on this. It is only happening when I look at my squeezebox touch.  My classic v3 and radio show tidal.  Also - it is not just Tidal that is hidden. It is most of my Apps.  Screenshots below show this:

Touch - My Apps
![Screenshot_20200303-171815_Squeezer](https://user-images.githubusercontent.com/45195982/75797020-cc18fa80-5d74-11ea-8a99-bb8382ff384d.jpg)

Classic V3 - My Apps
![Screenshot_20200303-171729_Squeezer](https://user-images.githubusercontent.com/45195982/75797218-22863900-5d75-11ea-9d6e-bfe5d88d14f5.jpg)

**Comment by kaaholst** [comment]
I have investigated this and I can't reproduce the issue. I installed Tidal and got the expected result as seen in the first 2 screenshots.
![device-2020-03-03-234159](https://user-images.githubusercontent.com/428304/75828218-a3f8be00-5dab-11ea-87c5-197acf4c4a75.png)
![device-2020-03-03-234251](https://user-images.githubusercontent.com/428304/75828225-a8bd7200-5dab-11ea-9b01-df3698ba2f7f.png)

I then created a free trail account and are able to browse TIDAL and play music as expected as shown in the third screenshot .
![device-2020-03-03-234947](https://user-images.githubusercontent.com/428304/75828236-ac50f900-5dab-11ea-8369-11116c781df2.png)
 
Squeezer works the same for all plugins and does nothing special for TIDAL. I therefore think this is a server or a player issue. I may well be different issues you are each experiencing. I suggest you look for answers in the slim forum and/or double check your setup in mysqueezebox.com.

**Comment by j-ame-s** [comment]
@kaaholst thanks for investigating. Are the old apk versions hosted anywhere or would I need to build it from git source?

I will try and get version 1.5.4 back on my phone as that worked with the server and players that I have with no issue.

**Comment by heniec** [comment]
Not only Tidal is invisible. I can't see in My Apps any app managed via mysqueezebox.com. Daphile managed plugins: Spotty and Youtube are ok.
I've googled and downloaded squeezer 1.5.4, deinstalled 2.0, installed 1.5.4. In older version all apps are ok.

**Comment by kaaholst** [comment]
Thanks for info.

This could be an issue with mysqueezebox.com managed plugins and players not synced with mysqueezebox.com
Please try to enable player sync in (local) server settings as shown in the image below and make sure that the player is enabled on the player settings tab in Tidal settings on mysqueezebox.com.

![image](https://user-images.githubusercontent.com/428304/76145899-51125580-608e-11ea-9e30-d01fa6b506ce.png)

@heniec 
To investigate this further could you please help me, what is "Daphile managed plugins"?

**Comment by j-ame-s** [comment]
@kaaholst good call. I just looked on mysqueezebox.com and the player not showing the tidal app is not registered (I got it from a friend so will try and get them to deregister it from mysqueezebox.com so I can add it to my account). I will update once to if this fixes the issue.

**Comment by j-ame-s** [comment]
@kaaholst My friend deregistered the player from his account and it then showed up in my account. I now see all apps for that player in squeezer. Issue fixed.

Strange that on the physical player I was able to see the app and play songs from it (it must have been using my local servers tidal details rather than the mysqueezebox.com ones).

**Comment by heniec** [comment]
@kaaholst, I use Daphile as my local music server. My settings for mysqueezebox.com on Daphile Advanced Settings are exactly as shown at your screen in your comment.
![image](https://user-images.githubusercontent.com/61412741/76152902-52ab4000-60c5-11ea-88ce-62a1cfa5e454.png)
I have Daphile installed on mini pc terminal, with external DAC. In Daphile settings, Player is the name if this DAC. On mysqueezebox.com I have no registered players.
Saying "Daphile managed plugins" I mean plugins available and enabled on Daphile server, with no need to adding them od mysqueezebox.com, i.e. Spotify or Youtube, working fine with old and new Squeezer version.

**Comment by przemnet** [comment]
Hmm... Just upgraded to 2.0.1 version and... I can see TIDAL again ;-)

THANK U ;-)

![Screenshot_2020-03-07-23-19-42-794_uk org ngo squeezer](https://user-images.githubusercontent.com/47325817/76153531-75415700-60cd-11ea-8d04-0b7dfbb699ad.jpg)
![Screenshot_2020-03-07-23-19-49-221_uk org ngo squeezer](https://user-images.githubusercontent.com/47325817/76153534-75d9ed80-60cd-11ea-9893-56975919ff4f.jpg)

**Comment by heniec** [comment]
@kaaholst, after some further investigation, it was definitely missing player issue. I had another account (with missing password - password reset function seems to be not working) on mysqueezebox.com, and my player was registered with this old account; after deleting old account with all registered players, my DAC player appeared on my current account on mysqueezeboc.com and Tidal appeared in My Apps in Squeezer 2.0.1.
Sorry for perturbation and thanks for attention...

**Comment by kaaholst** [comment]
Thanks for the feedback, I'm glad you got it working.
I have done some investigation into the issue, as to why TIDAL (or any mysqueezebox.com managed plugin) is visible on the physical player.
I am investigating the feasibility of doing the same in Squeezer so we can remove the dependency that the player is registered at mysqueezebox.com.

**Comment by error454** [comment]
I am having this issue on 2 android devices since updating to > 2.0.0. I actually thought it was a server issue so I upgraded/reimaged my LMS server and reconfigured everything. 

In my case I have 1 hardware player and 8 software players. They are all connected to mysqueezebox.com, all apps work fine if you control the device from the physics hardware remote or from the LMS web interface. But the app shows the screen from comment 2.

**Comment by kaaholst** [comment]
I looks like the player is registered to another mysqueezebox.com account than the account configured on your LMS. 
I suggest you look on mysqueezebox.com if your players show up in the players tab.
On your LMS check  that player sync is enabled and that it is connected to mysqueezebox.com with the same account (email address).
Hope this helps, otherwise please post back.

**Comment by error454** [comment]
> I looks like the player is registered to another mysqueezebox.com account than the account configured on your LMS.
> I suggest you look on mysqueezebox.com if your players show up in the players tab.
> On your LMS check that player sync is enabled and that it is connected to mysqueezebox.com with the same account (email address).

This is not the case for me. I'm the only owner of the hardware and only have a single account that has been working for the last 4 years. As a precaution though, I removed all devices from mysqueezebox.com and had LMS re-connect and add them again.

The issue persists and I believe it is 100% Squeezer since all of my mysqueezebox.com managed plugins work just fine in the OrangeSqueeze app on 2 devices but don't show up in Squeezer.

**Comment by F6DZP** [comment]
Hello,
I have the same issue, but only when I use it with my Transporter. When Squeezer drive my SqueezeRadio, I can see TIDAL application, when Squeezer drive my Duet  receiver, I can see the TIDAL application, When Squeezer drive my Transporter, serveral application disappear including TIDAL. If I use my Duet Controller, I can see all applications when I drive my Transporter, so I can access to TIDAL. I use Squeezer v2.0 and the issue remain. I vote 100% that it is a Squeezer issue.
driving SqueezeRadio:
![Screenshot_SqueezeRadio_Squeezer](https://user-images.githubusercontent.com/26501319/81036980-a00f1800-8e90-11ea-961a-567f7cca60ff.jpg)
driving Transporter:
![Screenshot_transporter_Squeezer](https://user-images.githubusercontent.com/26501319/81037012-bddc7d00-8e90-11ea-8797-95addfd5aa1e.jpg)

Thanks for Help

**Comment by F6DZP** [comment]
Last info:
I have discovered that my transporter was not in my Players list on mysqueezebox.com
I have now registered it and now fix this issue/ I can see all myApp using Squeezer.
I suppose that my controller take the list of MyApp from my local LMS server and that Squeezer takes some info about applications on my server (as it found 3 applications (Qobuz, ..) and takes more info about application on mysqueezebox.com... ?, as now it can shows my 10 applications.

Thanks for this very good and free application.
